flake.nix

```
@@ -276,6 +297,7 @@
             in pkgs.mkShell {
             buildInputs = with pkgs; [ cmake cudatoolkit util-linux gcc11 fenixStable ] ++ buildDeps;
             shellHook = ''
+              export CARGO_TARGET_DIR=$PWD/${CARGO_TARGET_DIR};
```

```suggestion
              export CARGO_TARGET_DIR="$PWD/${CARGO_TARGET_DIR}";
```

I think it might currently break for the brave individuals who put spaces into their paths.